### DISCUSSION TITLE
PicoCSS

### DISCUSSION THREAD

**Discussion by miles-po** [argyleink/open-props#374] [Ideas]
Have you considered joining efforts with PicoCSS? The two projects seem like a great fit and would make the two greater than the sum of their parts.

PicoCSS: a wonderful starting point for a project, focused strongly on HTML

Open Props: the ability to extend far beyond the basics cleanly and flexibly

Now imagine the variables used to customize between the two were the same! The shallowest learning curve imaginable combined with the cleanest CSS customization engine on the planet!

**Comment by argyleink** [comment]
I haven't but I could definitely see how the two would fit great!

**Reply by JamesJosephFinn** [reply]
@argyleink just dropping in this thread with an unrelated note: I wanted to express my gratitude for your distributing this project. I've imported the stylesheets into my Wordpress builder, Cwicly, and they have served to speed up my design workflow by orders of magnitude on this particular project. Peace and goodness to you sir.
